packages/astro/src/core/ssr/resolve-config-paths.ts

```diff
@@ -0,0 +1,94 @@
+import { tsconfigResolverSync, TsConfigResultSuccess } from 'tsconfig-resolver';
+import fs from 'fs'
+import { posix as ps } from 'path'
```

Comment:
Small nit: we've run into a lot of different personal preferences for referencing `path`. We landed on _always_ using `import path from 'path'` and then referencing `path.posix.[method]` for clarity/consistency.

Reply:
Fixed in 3d3de83bc55fffec686d073f9c6211d1ab301f01